src/app/options/qgscustomprojectionoptions.cpp

```diff
@@ -203,6 +206,9 @@ void QgsCustomProjectionOptionsWidget::pbnRemove_clicked()
     }
     mDefinitions.erase( mDefinitions.begin() + row );
   }
+
+  leName->setEnabled( false );
+  mCrsDefinitionWidget->setEnabled( false );
```

Comment:
If we remove an item but there is still other item in the list, *leName* and *mCrsDefinitionWidget* would remain disabled, even when we select an item in the list

we should either:
- On remove, set the current item as the first one like it's done in the constructor
- enable leName / mCrsDefinitionWidget on currentItemChanged

IMHA second solution is the obvious from a user point of view